perun-core/src/main/java/cz/metacentrum/perun/core/implApi/ISResponseData.java

```diff
@@ -0,0 +1,36 @@
+package cz.metacentrum.perun.core.implApi;
+
+import org.w3c.dom.Document;
+
+/**
+ * @author Vojtech Sassmann <vojtech.sassmann@gmail.com>
```

Comment:
Please describe class purpose in javadoc.

Reply:
Done.